### DISCUSSION TITLE
What is the difference between MesloLGS.ttf, MesloLGSDZ.ttf and so on

### DISCUSSION THREAD

**Discussion by riffraff** [ryanoasis/nerd-fonts#1289] [General]
Hey folks, forgive this naive question but I couldn't find the answer in the main readme or in the one for a downloaded font, and this is deeply confusing for me.

I downloaded a Meslo release and I expected it to contain a single .ttf file or a handful, but it contains 70+ :)

It seems that, other than variations for bold/italic/mono/proportional, we have a bunch more with these suffixes but I don't know what they mean. 

It might be a size related thing, but there seem to be too many variations for me to grasp it. 

So, what do they mean?

**Comment by Finii** [comment]
Lets see...

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/f9abfb4d-7664-4990-9cd8-d4f6767fd951)

_From https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/Meslo_

Following the link mentioned there https://github.com/andreberg/Meslo-Font:

```
The LG in Meslo LG stands for Line Gap. The idea behind it is to allow the user
to have some control over vertical spacing by choosing one of three line gap
variants: small, medium and large (Meslo LG S, Meslo LG M and Meslo LG L respectively).
```
```
Added a dotted zero version of Meslo LG which is called Meslo LG DZ.
```

**Comment by riffraff** [comment]
oh, I assumed it was a generic thing for all NF fonts, not just a peculiar Meslo thing, thanks!

**Reply by Finii** [reply]
Only these are Nerd Font abbrevs:
`NF` -> `Nerd Font`
`NFM` -> `Nerd Font Mono`
`NFP` -> `Nerd Font Propo`

For the meaning of these, see https://github.com/ryanoasis/nerd-fonts/discussions/1103#discussioncomment-4852120

The word `Mono` can come in different places, for example (contrived):
`JetBrains Mono Nerd Font` <-> `JetBrains Mono Nerd Font Mono`.
